backend/tracim_backend/tests/functional/test_live_messages.py

```diff
@@ -18,7 +18,12 @@ def big_html_document(workspace_api_factory, content_api_factory, session) -> Co
     workspace = workspace_api.create_workspace(label="Foobar")
     content_api = content_api_factory.get()
 
-    description = 2000000 * "a"
+    # NOTE: MySQL and MariaDB have a maximum of 65536 bytes for description,
+    # so the size is chosen accordingly
```

Comment:
Won't this effectively break Tracim 3.0 on MySQL / MariaDB as soon as someone writes a document bigger than 64 KiB?

Should we prevent people using MySQL from migrating to Tracim 3.0 until we sort this out?

Reply:
oh, by the way, you should put `2020-08-07 - SG` - in this comment.

Reply:
> Should we prevent people using MySQL from migrating to Tracim 3.0 until we sort this out?

The problem is already there.